ui/src/app/machines/views/MachineDetails/MachineTests/MachineTestsTable/ScriptRunTime/ScriptRunTime.tsx

```diff
@@ -0,0 +1,74 @@
+import { useEffect, useRef, useState } from "react";
+
+import { intervalToDuration, fromUnixTime } from "date-fns";
+import pluralize from "pluralize";
+
+import {
+  ScriptResultStatus,
+  ScriptResultEstimated,
+} from "app/store/scriptresult/types";
+import type { ScriptResult } from "app/store/scriptresult/types";
+
+type Props = {
+  scriptResult: ScriptResult;
+};
+
+// Add leading zeros to single digit numbers.
+const zeroPad = (time?: number) => `0${time || "0"}`.slice(-2);
+
+const getElapsedTime = (starttime: ScriptResult["starttime"]) => {
+  const { days, hours, minutes, seconds } = intervalToDuration({
+    start: starttime ? fromUnixTime(starttime) : Date.now(),
+    end: Date.now(),
+  });
+  const elapsed = [];
+  // Show the elapsed days if more than a day has elapsed.
+  if (days) {
+    elapsed.push(`${days} ${pluralize("day", days)}`);
```

Comment:
I think you can just use `elapsed.push(pluralize("day", days, true));`. The third argument prepends the string with the number.

Reply:
Thanks, I remember that now.